packages/admin/src/table/filterbar/filterBarPopoverFilter/FilterBarPopoverFilter.tsx

```diff
@@ -119,14 +50,17 @@ export function FilterBarPopoverFilter({
                 }}
                 initialValues={outerForm.getState().values}
             >
-                {({ form, values, handleSubmit }) => {
+                {({ form, values, handleSubmit, dirtyFields }) => {
+                    const countValue = calcNumberDirtyFields(values, form.getRegisteredFields());
+                    setHasFormValues(countValue > 0);
```

Comment:
don't create a state that replicates (parts) of another state (always have a single source of thruth)

if required, move the <div with class.root into `<Form` instead

Reply:
i solved it now in a different way :)